icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/util/ICUResourceBundleTest.java

```diff
@@ -717,10 +717,6 @@ public void TestLocaleDisplayNames() {
         }
 
         Set<String> localLangExceptions = new HashSet<>();
-        if (logKnownIssue("cldrbug:17001", "No localized language name for nmg, vmw")) {
-            localLangExceptions.add("nmg");
-            localLangExceptions.add("vmw");
-        }
```

Comment:
CLDR-17001 is done/fixed as of CLDR 44, but this test fails for language nmg.
The ticket only mentions language vmw, but this logKnownIssue was for this ticket and both languages.

https://github.com/unicode-org/icu/actions/runs/8147838298/job/22269422989?pr=2864
```
Error:  com.ibm.icu.dev.test.util.ICUResourceBundleTest.TestLocaleDisplayNames -- Time elapsed: 0.001 s <<< FAILURE!
java.lang.AssertionError: Could not get native localized language for nmg
	at org.junit.Assert.fail(Assert.java:89)
	at com.ibm.icu.dev.test.AbstractTestLog.errln(AbstractTestLog.java:50)
	at com.ibm.icu.dev.test.util.ICUResourceBundleTest.TestLocaleDisplayNames(ICUResourceBundleTest.java:738)
```

@DraganBesevic @pedberg-icu @srl295 @macchiati please advise.

Reply:
Thanks. I've just reverted this change.

Should we post another ticket for language nmg and change the reference?

Should I remove localLanExpections.add("vmw")?

Reply:
> Should I remove localLanExpections.add("vmw")?

Yes, please try that.

> Should we post another ticket for language nmg and change the reference?

Probably. Let's see if we can get a little more feedback.

@macchiati CLDR-17001 was about a language which was at basic but didn't even have an English display name. It looks like @pedberg-icu and/or @DraganBesevic used the same ticket for a second language that didn't get mentioned in the ticket and apparently didn't get fixed.

Reply:
Looking at the CLDR 45 coverage levels: https://www.unicode.org/cldr/charts/45/supplemental/locale_coverage.html
- vmw Target Level = none, Computed Level = basic
- nmg Target Level = none, Computed Level = none

So maybe CLDR doesn't really care yet, and ICU should be able to live with these exceptions?

But if nmg is not yet at basic level, then why did it get added to ICU?

Reply:
> Looking at the CLDR 45 coverage levels: https://www.unicode.org/cldr/charts/45/supplemental/locale_coverage.html
> 
> * vmw Target Level = none, Computed Level = basic
> * nmg Target Level = none, Computed Level = none
> 
> So maybe CLDR doesn't really care yet, and ICU should be able to live with these exceptions?
> 
> But if nmg is not yet at basic level, then why did it get added to ICU?

There was an issue where ICU was only looking at new locales and not looking at coverageLevels.txt (that's the data file that should be used).

Reply:
I submitted ICU-22681 “ICUResourceBundleTest.TestLocaleDisplayNames fails with ICU locales that are below CLDR basic coverage”